include/libtorrent/torrent_handle.hpp

```diff
@@ -840,7 +840,7 @@ namespace libtorrent
 		// already exist your torrent will be set on top of it and remaining once
 		// get posion + 1. If the given position is greater then queue it will be
 		// added on bottom
```

Comment:
this comment does not seem to recognize that every torrent (unless it's a seed) already has a queue position. when you change the queue position of a torrent, you're not only _adding_ a torrent to the queue, you're also _removing_ it from the queue first. i.e. your example of how other torrents get position + 1 is misleading because that's only true when you're moving a torrent up the queue (closer to the beginning). If you're moving the first torrent to the last position, every torrent in between will get the position - 1.

This is why the way I suggested phrasing it seems more accurate (and succinct).